it/uk/gov/hmrc/agentmappingfrontend/controllers/MappingControllerISpec.scala

```diff
@@ -63,263 +71,78 @@ class MappingControllerISpec extends BaseControllerISpec with AuthStubs {
       val request = FakeRequest(GET, "/agent-mapping/sign-in-required")
       val result = callEndpointWith(request)
       status(result) shouldBe 200
-      checkHtmlResultContainsMsgs(result, "start.not-signed-in.title")
+      checkHtmlResultContainsMsgs(result, "start.not-signed-in.title", "button.signIn")
     }
 
-    "303 the /start page when user has HMRC-AS-AGENT/ARN" in {
+    "303 the /start page when user has HMRC-AS-AGENT/ARN and 'Sign in with another account' button holds idReference to agent's ARN" in {
       givenUserIsAuthenticated(mtdAsAgent)
       val request = FakeRequest(GET, "/agent-mapping/sign-in-required")
       val result = callEndpointWith(request)
-      redirectLocation(result) shouldBe Some(routes.MappingController.start().url)
+      redirectLocation(result) shouldBe Some(routes.MappingController.start().url) //TODO APB-2866
```

Comment:
APB-2866 is flagged as Done. Is something missing here?

Reply:
was created yesterday, Ill ask Ian if were still doing this